packages/lib/src/spatial-navigation/context/DeviceContext.tsx

```diff
@@ -42,6 +43,7 @@ export const SpatialNavigationDeviceTypeProvider = ({ children }: DeviceProvider
   const getScrollingIntervalId = useCallback(() => scrollingId.current, []);
 
   useEffect(() => {
+    if (Platform.OS === 'web') return;
```

Comment:
I don't get it, shouldn't it be the opposite?
If the platform is not web, I don't care for the rest of the logic?

Reply:
Fixed :)